ui/src/helpers/get-status-class.ts

```diff
@@ -31,6 +31,8 @@ export const VulnerabilityRatings = {
   NONE: 0,
 } as const;
 
+export type VulnerabilityRating = keyof typeof VulnerabilityRatings;
```

Comment:
As (I believe) this type is now used where type is needed, I'd rename `VulnerabilityRatings` to `vulnerabilityRatings` to follow conventions more closely.

Reply:
No, sry, I misunderstood your comment. Will fix.